index.js

```diff
@@ -18,6 +18,17 @@ export default function (options) {
   const filter = createFilter(include || '**/*.svg', exclude);
   return {
     name: 'vue-inline-svg',
+    resolveId: function (source) {
+      // Metches absolute paths
+      // of svgs in node_modules
+      // that are skipped by default in rollup
+      var regexp = /^(?!\.)\S+\.svg$/gi;
+      if (source.match(regexp)) {
+        var rootDir = process.cwd();
+        return { id: rootDir + '/node_modules/' + source, external: false };
```

Comment:
Could you change this to use node's path module resolve?

Reply:
Good point, thank you!

I've updated the PR.

Reply:
Hi @e-e-e any news about this PR?

Reply:
My apologies I will merge now.

Reply:
Thanks a lot!